### PR TITLE
Use apidoc directive in mailboxes.md (#22904)

### PR DESCRIPTION
**Pull request by Captain1653**
References #22904

### CODE REVIEW

File: akka-docs/src/main/paradox/mailboxes.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ The type of the queue in the mailbox created for an actor will be checked agains
 ### Requiring a Message Queue Type for a Dispatcher
 
 A dispatcher may also have a requirement for the mailbox type used by the
-actors running on it. An example is the @apidoc[BalancingDispatcher] which requires a
+actors running on it. An example is the `BalancingDispatcher` which requires a
```
Comment (Captain1653) [inline akka-docs/src/main/paradox/mailboxes.md:77]: I removed the link. `BalancingDispatcher` is a private class, it's scaladoc/javadoc is not accessible

File: akka-docs/src/main/paradox/mailboxes.md [hunk 2]
```diff
@@ -214,4 +214,4 @@ Java
     @@@
 
 will probably fail; you will have to allow for some time to pass and retry the
-check à la `TestKit.awaitCond`.
+check à la @scala[@scaladoc[TestKit.awaitCond](akka.testkit.TestKit#awaitCond(p:=%3EBoolean,max:scala.concurrent.duration.Duration,interval:scala.concurrent.duration.Duration,message:String):Unit)]@java[@javadoc[TesKit.awaitCond)](akka.testkit.javadsl.TestKit#awaitCond(java.time.Duration,java.time.Duration,java.lang.String,java.util.function.Supplier))].
```
Comment (Captain1653) [inline akka-docs/src/main/paradox/mailboxes.md:217]: I'm not sure about the links. Can you confirm?

Comment (raboof) [reply akka-docs/src/main/paradox/mailboxes.md:217]: That looks fine to me, except for the Java label

```suggestion
check à la @scala[@scaladoc[TestKit.awaitCond](akka.testkit.TestKit#awaitCond(p:=%3EBoolean,max:scala.concurrent.duration.Duration,interval:scala.concurrent.duration.Duration,message:String):Unit)]@java[@javadoc[TestKit.awaitCond](akka.testkit.javadsl.TestKit#awaitCond(java.time.Duration,java.time.Duration,java.lang.String,java.util.function.Supplier))].
```

**Comment by Captain1653** [comment]
@raboof Thank you for the review. Something bad happen :(  . Is it my fault?

**Comment by raboof** [comment]
> @raboof Thank you for the review. Something bad happen :( . Is it my fault?

No, I think GitHub was having a hiccup (though I don't see anything at https://www.githubstatus.com/ )

**Comment by Captain1653** [comment]
I saw it, but now it works
Снимок экрана 2021-12-06 в 18 31 27

**Review by octonato — APPROVED** [review]
LGTM

**Review by raboof — APPROVED** [review]

**Pull request merged**